### ISSUE TITLE
Small bug in create tower

### ISSUE THREAD

**Issue by shanuv** [tensorflow/ranking#361]
In lines 70 and 76 of tensorflow_ranking/python/keras/layers.py, the code snippet should have **kwargs as input to `tf.keras.layers.Dense` 

`model = tf.keras.Sequential(name=name)
  # Input batch normalization.
  if input_batch_norm:
    model.add(tf.keras.layers.BatchNormalization(momentum=batch_norm_moment))
  for layer_width in hidden_layer_dims:
    model.add(tf.keras.layers.Dense(units=layer_width), **kwargs)
    if use_batch_norm:
      model.add(tf.keras.layers.BatchNormalization(momentum=batch_norm_moment))
    model.add(tf.keras.layers.Activation(activation=activation))
    if dropout:
      model.add(tf.keras.layers.Dropout(rate=dropout))
  model.add(tf.keras.layers.Dense(units=output_units), **kwargs)
  return model
`
